alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/metadataExtractors/IPTCMetadataExtractor.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * #%L
+ * Alfresco Transform Core
+ * %%
+ * Copyright (C) 2005 - 2021 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * -
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ * -
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * -
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * -
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.transformer.metadataExtractors;
+
+import java.io.IOException;
+
+import org.alfresco.transform.exceptions.TransformException;
+import org.alfresco.transformer.tika.parsers.ExifToolParser;
+import org.apache.tika.exception.TikaException;
+import org.apache.tika.parser.Parser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class IPTCMetadataExtractor extends AbstractTikaMetadataExtractor
+{
+
+    private static final Logger logger = LoggerFactory.getLogger(IPTCMetadataExtractor.class);
+
+    public IPTCMetadataExtractor() 
+    {
+        super(logger);
+    }
+
+    @Override
+    protected Parser getParser() {
+        try {
+            return new ExifToolParser();
+        } catch (IOException | TikaException e) {
+            throw new TransformException(500, "Error creating IPTC parser");

```

Comment:
We should also log the exception message as an ERROR in the logs / service console.

Reply:
done 👍